src/index.html

```diff
@@ -11,7 +11,153 @@
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>HTML Form</h1>
+    <form
+    action="https://mate-academy-form-lesson.herokuapp.com/create-application"
+    method="post"
+    name="personalInfo"
+    >
+      <fieldset class="personal__info">
+        <legend>Personal information:</legend>
+        <label class = "personal__input" for="surname">Surname:
+          <input
+          id = "surname"
+          type="text"
+          autocomplete="off"
+          required
+          >
```

Comment:
```suggestion
    <form
      action="https://mate-academy-form-lesson.herokuapp.com/create-application"
      method="post"
      name="personalInfo"
    >
      <fieldset class="personal__info">
        <legend>Personal information:</legend>
        <label class = "personal__input" for="surname">
          Surname:
          <input
            id = "surname"
            type="text"
            autocomplete="off"
            required
          >
```
pay attention to the checklist and fix your code according to it